drivers/serial/Kconfig.mcux_iuart

```diff
@@ -0,0 +1,21 @@
+# MCUXpresso SDK IUART
+
+# Copyright (c) 2019, Manivannan Sadhasivam <mani@kernel.org>
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_MCUX_IUART
+	bool "MCUX IUART driver"
+	depends on HAS_MCUX && CLOCK_CONTROL
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	help
+	  Enable the MCUX IUART driver.
+
+if UART_MCUX_IUART
+
+config UART_MCUX_IUART_4
+	bool "UART 4"
+	help
+	  Enable UART 4.
```

Comment:
We've recently removed driver instance Kconfigs, please see 5a149dfca697e2716d0e96770be1358c9ec0e618